ethereum/contracts/pyth/PythUpgradable.sol

```diff
@@ -65,11 +67,25 @@ contract PythUpgradable is Initializable, OwnableUpgradeable, UUPSUpgradeable, P
         PythSetters.setValidTimePeriodSeconds(newValidTimePeriodSeconds);
     }
 
+    // Privileged function to update the governance emitter
```

Comment:
this function exists so you can do the migration from the current owner -> governance?

Reply:
if so, maybe leave a comment that this function can be deleted once the migration happens

Reply:
All the privileged functions above are like this. I'm not sure we can remove it with the current truffle deployment process though. (every contract on a new chain is deployed with an owner without governance emitter and then all the migration steps happen)